Shop.API/ActionFilters/HttpGlobalExceptionFilter.cs

```diff
@@ -0,0 +1,51 @@
+﻿using FluentValidation;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Shop.API.ActionFilters.Extensions;
+using Shop.BLL.Common.Exceptions.BadRequestExceptions;
+using Shop.BLL.Common.Exceptions.NotFoundExceptions;
+
+namespace Shop.API.ActionFilters;
+
+public class HttpGlobalExceptionFilter(IHostEnvironment environment) : IAsyncExceptionFilter
```

Comment:
it is better to use a middleware rather than a filter

Reply:
Why so?